samples/Azure.AI.FormRecognizer/Customization/CopyContent.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using Azure.Core;
+
+namespace Azure.AI.FormRecognizer.Models
```

Comment:
Why not use autorest directive [rename-model](https://github.com/Azure/autorest/blob/6caa31c18ff03173a8f4a945bfe0306c94463aa8/docs/generate/built-in-directives.md#rename-model)? For mgmt SDK, you can even use [rename-mapping](https://github.com/Azure/azure-sdk-for-net/blob/8cf16de988e6c00b510d8c7cea7bbf6d3c0afd8f/sdk/maps/Azure.ResourceManager.Maps/src/autorest.md?plain=1#L21).

Reply:
well this is the fastest way - other approaches need to find their swagger

Reply:
What's the advantage?

Reply:
I prefer to using customized code, in the feature, we may migrate to typespec, and swagger is deprecated step by step. So it may introduce directive